### PR TITLE
fix: patch telemetry

### PR DESCRIPTION
**Pull request by code-asher**
This seems to have been dropped in the rewrite.

### CODE REVIEW

File: patches/telemetry.diff [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+ 						scope: vscodeBase + '/',
+Index: code-server/lib/vscode/src/vs/workbench/services/telemetry/browser/telemetryService.ts
+===================================================================
+--- code-server.orig/lib/vscode/src/vs/workbench/services/telemetry/browser/telemetryService.ts
++++ code-server/lib/vscode/src/vs/workbench/services/telemetry/browser/telemetryService.ts
+@@ -25,6 +25,7 @@ class WebAppInsightsAppender implements
+ 	private _telemetryCache: { eventName: string; data: any }[] = [];
+
+ 	constructor(private _eventPrefix: string, aiKey: string) {
++		console.log("halp")
```
Comment (jsjoeio) [inline patches/telemetry.diff:188]: typo? "help"

File: patches/telemetry.diff [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+ 		const endpointUrl = 'https://vortex.data.microsoft.com/collect/v1';
+ 		import('@microsoft/applicationinsights-web').then(aiLibrary => {
+ 			this._aiClient = new aiLibrary.ApplicationInsights({
+@@ -74,6 +75,7 @@ class WebAppInsightsAppender implements
+ 		data = { ...data.properties, ...data.measurements };
+
+ 		// undefined assertion is ok since above two if statements cover both cases
++		console.log("are we here maybe?")
```
Comment (jsjoeio) [inline patches/telemetry.diff:196]: nice, I usually write the same thing lol

**Review by jsjoeio — CHANGES_REQUESTED** [review]
Typo in your `console.log` 😉

File: patches/telemetry.diff (hunk 1 above)
Comment (code-asher) [reply patches/telemetry.diff:188]: looooool oops

File: patches/telemetry.diff (hunk 2 above)
Comment (code-asher) [reply patches/telemetry.diff:196]: hahaha it would seem I forgot to refresh the patch after removing :rofl:

File: patches/telemetry.diff (hunk 1 above)
Comment (code-asher) [reply patches/telemetry.diff]: halp is the cute version of help ;)

File: patches/telemetry.diff (hunk 2 above)
Comment (code-asher) [reply patches/telemetry.diff]: Er I mean this is a feature :eyes:

**Review by jsjoeio — APPROVED** [review]
🎉

**Pull request merged**